.github/actions/gradle-setup/action.yml

```diff
@@ -18,6 +18,13 @@ runs:
         path: RegistrationService
         ref: 374c14bcca23ddb1dcd7476a27264510e54de7fa
 
+    - name: Checkout Identity Hub
+      uses: actions/checkout@v2
+      with:
+        repository: eclipse-dataspaceconnector/IdentityHub
+        path: IdentityHub
```

Comment:
wouldn't that resolve to `eclipse-dataspaceconnector/IdentityHub/IdentityHub`?

Reply:
The variable "Path" refers to the path where the repository is checked out locally

Reply:
ahh, right.